backend/app/router/endpoints/quizes.py

```
@@ -0,0 +1,69 @@
+from fastapi import APIRouter, HTTPException
+from typing import List, Tuple, Union
+from app.pydantic_models.quiz import Quiz
+from app.pydantic_models.question import Question
+from app.pydantic_models.multiselect_question import MultiSelectQuestion
+from pydantic import BaseModel
+
+router = APIRouter()
+
+# In-memory store of quizzes
+quizes = {}
+next_quiz_id = 1  # Start from 1
```

Pylint (3.3.7)

[convention] Constant name "next_quiz_id" doesn't conform to UPPER_CASE naming style

(C0103)